### PR TITLE
Host step 3

### PR DESCRIPTION
**Pull request by deetz99**
*Issue:*

- bcgov/entity/issues/24719

*Description of changes:*
- step 3 doc upload stuff
- validate docs/business licence

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: strr-host-pm-web/app/components/form/DefineYourRental/UnitRequirements/Error.vue [hunk 1]
```diff
@@ -63,7 +63,7 @@ function handleContinueApp () {
                 >
                   <template #linkAllRules>
                     <a
-                      href=""
+                      href="https://www2.gov.bc.ca/gov/content/housing-tenancy/short-term-rentals"
```
Comment (thorwolpert) [inline strr-host-pm-web/app/components/form/DefineYourRental/UnitRequirements/Error.vue:66]: Can we put those as configs?
or in the i18n?

**Review by thorwolpert — APPROVED** [review]
Looks good to me, just wondering if we should pull the CMS links out of the pages?

File: strr-host-pm-web/app/components/form/DefineYourRental/UnitRequirements/Error.vue (hunk 1 above)
Comment (deetz99) [reply strr-host-pm-web/app/components/form/DefineYourRental/UnitRequirements/Error.vue:66]: yeah sure I can do that

Comment (deetz99) [reply strr-host-pm-web/app/components/form/DefineYourRental/UnitRequirements/Error.vue]: fixed

**Review by thorwolpert — APPROVED** [review]
thanks

**Pull request merged**